infrastructure/comms/src/connection/message.rs

```diff
@@ -118,4 +159,30 @@ mod test {
             _ => panic!("Unexpected MessageError {:?}", error),
         }
     }
+
+    #[test]
+    fn test_ser_des() {
+        let version = 0;
+        let mut rng = rand::OsRng::new().unwrap();
+        let k = RistrettoSecretKey::random(&mut rng);
+        let p = RistrettoPublicKey::from_secret_key(&k);
+        let source = p;
+        let dest: NodeDestination<RistrettoPublicKey> = NodeDestination::Unknown;
+        let signature = vec![0];
+        let flags = 0; // IdentityFlags::ENCRYPTED;
```

Comment:
I think you want `IdentityFlags::empty()` here

Reply:
I actually wanted`IdentityFlags::ENCRYPTED` there, but I removed the flags struct in ln 66 for testing porposes. I never put it back. Hence the 0 there. But its fixed now.